src/ibidem.rs

```diff
@@ -71,15 +73,30 @@ impl<P: syn::parse::Parse> std::ops::Deref for ParseWrapper<P> {
 
 /// While it is convenient to be able to "pass through" TokenStreams
 /// unperturbed, or to interpret TokenStreams via syn::parse::Parse.
```

Comment:
I'm not sure where I was going with this. Perhaps

```suggestion
/// While it is convenient to be able to "pass through" TokenStreams
/// unperturbed, or to interpret TokenStreams via syn::parse::Parse.
/// that breaks serde's model.,
```

Reply:
I guess we now _are_ passing through TokenStreams unperturbed so I'll reword this slightly.